spec/auth_spec.rb

```diff
@@ -1,23 +1,23 @@
-require 'capybara/rspec'
+# require 'capybara/rspec'
 
 RSpec.describe 'Sign in', type: :system do
-  before :each do
-    User.create(name: 'Rock', email: 'rock@star.com', password: 'tomtomtown')
-  end
+  #   before :each do
+  #     User.create(name: 'Rock', email: 'rock@star.com', password: 'tomtomtown')
+  #   end
 
-  it 'signs me in' do
-    visit '/users/sign_in'
-    sleep(5)
+  #   it 'signs me in' do
+  #     visit '/users/sign_in'
```

Comment:
- [ ] Delete all the commented-out pieces of code in all the files where they exist.

Reply:
done